Dockerfile

```diff
@@ -63,6 +63,9 @@ COPY --from=backend /backend-build/bytebase /usr/local/bin/
 # Directory to store the data, which can be referenced as the mounting point.
 RUN mkdir -p /var/opt/bytebase
 
+# Copy online demo starting scripts
```

Comment:
The comment is less accurate, what if we later add another script which is not relevant to demo?

So here we either just copy that demo script, or we form the comment in a concise way:

# Copy utility scripts, we have
# - Demo script to launch Bytebase in readonly demo mode